attributes/default.rb

```diff
@@ -58,6 +58,18 @@
 default['cfncluster']['intelmpi']['version'] = '2019.7.217'
 default['cfncluster']['intelmpi']['modulefile'] = "/opt/intel/impi/#{node['cfncluster']['intelmpi']['version']}/intel64/modulefiles/mpi"
 
+# Arm Performance Library
+default['cfncluster']['armpl']['version'] = '20.2.1'
+default['cfncluster']['armpl']['gcc']['version'] = '9.3'
+default['cfncluster']['armpl']['gcc']['url'] = "http://ftp.gnu.org/gnu/gcc/gcc-#{node['cfncluster']['armpl']['gcc']['version']}.0/gcc-#{node['cfncluster']['armpl']['gcc']['version']}.0.tar.gz"
+if node['platform'] == 'ubuntu'
+  default['cfncluster']['armpl']['platform'] = 'Ubuntu-16.04'
+  default['cfncluster']['armpl']['url'] = "https://developer.arm.com/-/media/Files/downloads/hpc/arm-performance-libraries/20-2-1/Ubuntu16.04/arm-performance-libraries_20.2.1_Ubuntu-16.04_gcc-#{node['cfncluster']['armpl']['gcc']['version']}.tar"
+else
+  default['cfncluster']['armpl']['platform'] = 'RHEL-8'
+  default['cfncluster']['armpl']['url'] = "https://developer.arm.com/-/media/Files/downloads/hpc/arm-performance-libraries/20-2-1/RHEL8/arm-performance-libraries_20.2.1_RHEL-8_gcc-#{node['cfncluster']['armpl']['gcc']['version']}.tar"
```

Comment:
Just to confirm, is the package for RHEL8 also valid for AL2?

Reply:
yes, this package is valid for AL2